### PR TITLE
Fix policies incorrectly defaulting to object level action when a property is specifically called out

### PR DESCRIPTION
**Pull request by CarsonF**
Here's an example of something being ignored.
https://github.com/SeedCompany/cord-api-v3/blob/ad02252edfda9eb300e3c15282aceabac3fb661e/src/components/authorization/policies/by-role/consultant-manager.policy.ts#L20-L20
It was falling back to organization object level _read_ instead of using the absence of actions explicitly called out on the `address` property.

### CODE REVIEW

**Review by bryanjnelson — APPROVED** [review]
🚢

File: src/components/authorization/policies/by-role/financial-analyst.policy.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ import {
             'financialReportPeriod',
             'financialReportReceivedAt',
           )
-          .when(member).edit,
+          .read.when(member).edit,
```
Comment (bryanjnelson) [inline src/components/authorization/policies/by-role/financial-analyst.policy.ts:62]: Edit implies read, correct?  But in this case we want no member restrictions on reads so that's why the change?

File: src/components/authorization/policies/by-role/regional-director.policy.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { member, Policy, Role, sensMediumOrLower } from '../util';
 
   r.Partnership.read,
   r.Project.when(member).edit.specifically(
-    (p) => p.rootDirectory.when(sensMediumOrLower).read,
+    (p) => p.rootDirectory.edit.when(sensMediumOrLower).read,
```
Comment (bryanjnelson) [inline src/components/authorization/policies/by-role/regional-director.policy.ts:9]: So here the root directory was not editable, but now you are making it editable?

**Review by bryanjnelson — APPROVED** [review]

File: src/components/authorization/policies/by-role/financial-analyst.policy.ts (hunk 1 above)
Comment (CarsonF) [reply src/components/authorization/policies/by-role/financial-analyst.policy.ts:62]: Exactly. We call out these properties because we want to _add_ **edit** access in certain cases. But because they are called out, the object level **read** is ignored.
So we need to "restore" that before widening the scope.

File: src/components/authorization/policies/by-role/regional-director.policy.ts (hunk 2 above)
Comment (CarsonF) [reply src/components/authorization/policies/by-role/regional-director.policy.ts:9]: Logically there's no change here. This was added to restore previous functionality with this bug fix.

We want `rootDirectory` to be editable when you are a member.
Which the `.edit` does since it takes the closest previously defined condition.

Like above, we are additionally adding read access for low/med sens projects even if you are not a member.
But because we called that out we also need to restore the object-level permission on this property.

**Comment by CarsonF** [comment]
As two different use cases are revealing, it could be worth it to have another method.
```ts
// edit everything, but this one prop.
X.edit.narrowProps(p => p.prop.read)
```
```ts
// read everything including prop, and also allow editing prop when member.
X.read.widenProps(p => p.prop.when(member).edit)
```

**Comment by bryanjnelson** [comment]
> As two different use cases are revealing, it could be worth it to have another method.
> 
> ```ts
> // edit everything, but this one prop.
> X.edit.narrowProps(p => p.prop.read)
> ```
> 
> ```ts
> // read everything including prop, and also allow editing prop when member.
> X.read.widenProps(p => p.prop.when(member).edit)
> ```

This would probably make things more clear, yes.
